MusicCrawler/Program.cs

```diff
@@ -17,15 +21,22 @@ async Task PrintLibrariesAndRecentlyAdded()
     }
 }
 
+async Task ManuallyVerifySpotifyApi()
+{
+    var result = await spotifyRepo.Recommendations("4NHQUGzhtTLFvgF5SZesLK");
```

Comment:
Is this meant to be querying for the artist `Tove Lo`?
https://open.spotify.com/artist/4NHQUGzhtTLFvgF5SZesLK

Eventually we'll probably want to refactor this to abstract away this ID and take in the artist name, and have the repo look up the spotify specific ID.  But this probably fine for now